sitemap.js

```diff
@@ -34,7 +35,7 @@ const schedule = (function () {
         let timeout;
 
         if (immediate) {
-            timeout = 4;
+            timeout = ms('1s');
```

Comment:
`1s` is not immediate. If we want 1s delay, why then we specify `immediate` flag?

Reply:
May be it worth to comment why 4 is regarded as immediate 😃 (really, does not matter as long as it runs very soon). Other option would be to invoke `run` immediately, and set timeout to whatever the next sheduled run should be. This will be refactored anyway when we have job scheduler.

Reply:
Reverted this one.